html/ui/html/modals/escrow.html

```diff
@@ -98,7 +98,7 @@ <h4 class="modal-title" data-i18n="escrow_decision">Escrow Decision</h4>
           <div class="form-group">
             <label data-i18n="Escrow">Escrow</label>
             <input type="text" name="escrow" class="form-control" id="escrow_decision_escrow" />
-            <select class="form-control" name="decision" id="escrow_decision_decision">
+            <select class="form-control" name="decisionType" id="escrow_decision_decision">
```

Comment:
Did you try this out by chance? Hard to tell if it was done right end to end.

Reply:
oops, that would've been caught up in an intellij refactor